### ISSUE TITLE
lua rpm_vercmp error message if second argument is broken

### ISSUE THREAD

**Issue by AvoidMe** [rpm-software-management/rpm#2165]
I found a small issue while was playing with lua in rpm spec:
```
➜  rpm --eval "%{lua:rpm.interactive()}"

RPM Interactive Lua 5.4 Interpreter
> print(rpm.vercmp('1', '1'))
0
> print(rpm.vercmp('', '1'))         
[string "<lua>"]:1: bad argument #1 to 'vercmp' (invalid version )
> print(rpm.vercmp('1', '')) 

> 
```
If first argument is wrong in `rpm.vercmp()` - meaningful error message as expected
if second argument is wrong - nothing happens and nil returned

**Comment by AvoidMe** [comment]
I submited a pr for this issue: https://github.com/rpm-software-management/rpm/pull/2163

**Comment by pmatilai** [comment]
Fixed by #2163
